staging/src/k8s.io/apiserver/pkg/authentication/authenticatorfactory/delegating.go

```diff
@@ -54,6 +55,9 @@ type DelegatingAuthenticatorConfig struct {
 	APIAudiences authenticator.Audiences
 
 	RequestHeaderConfig *RequestHeaderConfig
+
+	// WebhookRetryBackoff specifies the backoff parameters of webhook retry
```

Comment:
Finally found the spot I care about.  maybe tell me why I would want to set this. Our example use-case could be helpful.

Also, I expected to see a `WithCustomRetryBackoff` function like we use for `ResourceBuilderFlags`.

Reply:
@deads2k I added the `WithCustomRetryBackoff` function in `DelegatingAuthenticationOptions` - https://github.com/kubernetes/kubernetes/pull/95705/commits/5df6f2d43697b3076438f46400b3bdfeff0b8d84#diff-249fc1fe96479271dbe3cbd4847642b338cfc33a10f1f033fa606da704462320R213-R217. does that sound correct?